### ISSUE TITLE
Porting the library to dnxcore50

### ISSUE THREAD

**Issue by follesoe** [GeoJSON-Net/GeoJSON.Net#54]
Have anyone made an attempt to port the library to CoreCLR/dnxcore50?

I'm building an ASP.NET 5/ASP.NET Core project and would like to use this library. I'd might have a stab at porting it. Opening this issue to discuss if anyone have attempted it so far.

**Comment by jbattermann** [comment]
Hi Jonas,

to be honest I've played with the thought of making this library properly portable & more flexible (different serializers besides JSON.Net etc) but I am personally quite unhappy regarding the state it's currently in overall (the quality has deteriorated over the years) and would rather start a proper rewrite but I simply don't have the time currently nor does any of my own projects or clients require or justify major time investments in it and therefore even thought about giving up the project 'for adoption' ;)

So basically IF you do want to start porting it over and IF / whenever you might be done send over a PR, I'd really gladly merge it in - and if you want to improve / rewrite certain unpleasant existing corners that you might find along the way and might want to improve, don't hesitate and just do it (just saying - I *know* there are a few dark corners).

Cheers,
-J

**Comment by hanssens** [comment]
@jbattermann The PR in the reference above this should fix this. Details in the PR itself. 
Hope it helps ;)

**Comment by matt-lethargic** [comment]
The library now supports .Net Standard 1.1, this covers .Net Core
[https://docs.microsoft.com/en-us/dotnet/standard/library](https://docs.microsoft.com/en-us/dotnet/standard/library)

Closing issue
